udp/client/clientconn.go

```diff
@@ -672,9 +672,9 @@ func (cc *ClientConn) Process(datagram []byte) error {
 			return
 		}
 
-		// send message with confirmation
-		w.response.SetType(udpMessage.Confirmable)
-		w.response.SetMessageID(cc.getMID())
+		// send piggybacked response
+		w.response.SetType(udpMessage.Acknowledgement)
```

Comment:
Hi. We need to handle when we send `udpMessage.Acknowledgement`. With this response, it always sends `udpMessage.Acknowledgement` but for blockwise transfer, we want to send a confirmation message.
So change this line to something like:
```
w.response.SetType(udpMessage.Confirmable)
if reqType == udpMessage.Confirmable {
  w.response.SetType(udpMessage.Acknowledgement)
} ;
```

Reply:
Hi, thanks for the review and sorry for my long absence. I have incorporated your request and hope it is fine now :)